assets/js/editor.js

```diff
@@ -129,6 +129,28 @@ $gp.editor = (
 					} else {
 						$gp.editor.save( target.parents( 'tr.editor' ).find( 'button.ok' ) );
 					}
+				} else if ( 13 === e.keyCode && e.ctrlKey ) { // Ctrl-Enter = Copy original.
+					target = $( e.target );
+
+					original = $( '.editor:visible' ).find( '.original' );
+
+					target.val( original.text() );
+				} else if ( 107 === e.keyCode && e.ctrlKey ) { // Ctrl-+ = Approve.
```

Comment:
I think we'll never avoid all shortcuts used by every os or user, we already hijack PgUp/PgDn.  This is also only active within the editor box, so I don't see it as a big deal.